PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/RiderAzureUtils.kt

```diff
@@ -22,8 +22,34 @@
 
 package org.jetbrains.plugins.azure
 
+import org.apache.http.client.utils.URIUtils
+import java.net.URI
+import java.util.*
+
 fun String?.orWhenNullOrEmpty(another: String): String = if (this.isNullOrEmpty()) {
     another
 } else {
     this
+}
+
+fun String?.isValidUrl(): Boolean {
+    if (this.isNullOrEmpty()) return false
+
+    return try {
+        val uri = URI.create(this)
+        URIUtils.extractHost(uri) != null
+    } catch (_: Exception) {
+        false
+    }
+}
+
+fun String?.isValidUUID(): Boolean {
```

Comment:
Could you please cover this with some tests

Reply:
Fixed.